ftplugin/gitcommit.vim

```diff
@@ -1,6 +1,7 @@
-if g:lazygit_use_neovim_remote && executable("nvr")
+if g:lazygit_opened && g:lazygit_use_neovim_remote && executable("nvr")
     augroup lazygit_neovim_remote
       autocmd!
       autocmd WinLeave <buffer> :LazyGit
+      autocmd WinLeave <buffer> :setglobal lazygit_opened=0
```

Comment:
I've never used `setglobal` before maybe `let g:lazygit_opened=0` will work better here

Reply:
Ah just did some digging, I think `setglobal` is for setting *options* not for adding values to the the global object.

Reply:
also just as an fyi I don't think you need the `:` here. Think these will work as `<buffer> LazyGit` etc :shrug: